book/src/language-features/strings.md

```diff
@@ -13,7 +13,7 @@ select x = 'hello world'
 ```
 
 To quote a string containing quotes, either use the "other" type of quote, or
-use three-or-more quotes, and close with the same number.
+use 3, 4, 5 or 6 quotes, and close with the same number.
```

Comment:
Ah interesting. This seems fine. Because it's easier to make for 3-6 than for `n`?

Reply:
Because I don't know how to implement for `n` :D

https://github.com/PRQL/prql/pull/1818/files#diff-ae47c7322a8e7920d5819e0c5a9d2fdf1210e576e808f91a1f4ee72d013e4c2dR267-R283